types/src/on_chain_config/consensus_config.rs

```diff
@@ -47,21 +50,38 @@ impl OnChainConsensusConfig {
             OnChainConsensusConfig::V1(config) | OnChainConsensusConfig::V2(config) => {
                 &config.proposer_election_type
             },
+            _ => unimplemented!("method not supported"),
         }
     }
 
     pub fn quorum_store_enabled(&self) -> bool {
         match &self {
             OnChainConsensusConfig::V1(_config) => false,
             OnChainConsensusConfig::V2(_config) => true,
+            OnChainConsensusConfig::DagV1(_) => false,
```

Comment:
Why disable quorum store with DAG?

Reply:
oh it was panicking before, also I think it comes with duplicate txn since we haven't disabled the broadcast